app/src/main/res/layout-sw600dp-land/add_profile_activity.xml

```diff
@@ -0,0 +1,216 @@
+
+  <data>
+
+    <import type="android.view.View" />
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.profile.AddProfileViewModel" />
+  </data>
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include
+      android:id="@+id/add_profile_activity_toolbar"
+      layout="@layout/toolbar" />
+
+    <FrameLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent">
+
+      <ScrollView
+        android:id="@+id/add_profile_activity_scroll_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/addProfileBackground">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:paddingStart="232dp"
+          android:paddingTop="52dp"
+          android:paddingEnd="232dp"
+          android:paddingBottom="224dp">
+
+          <com.jackandphantom.circularimageview.CircleImage
+            android:id="@+id/add_profile_activity_user_image_view"
+            android:layout_width="120dp"
+            android:layout_height="120dp"
+            android:clickable="true"
+            android:focusable="true"
+            android:src="@drawable/ic_default_avatar"
+            app:add_shadow="true"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:shadow_radius="8" />
+
+          <ImageView
+            android:id="@+id/add_profile_activity_edit_user_image_view"
+            android:layout_width="48dp"
+            android:layout_height="48dp"
+            android:paddingStart="12dp"
+            android:paddingTop="12dp"
+            android:src="@drawable/rounded_white_background_with_shadow"
+            app:layout_constraintBottom_toBottomOf="@+id/add_profile_activity_user_image_view"
+            app:layout_constraintEnd_toEndOf="@+id/add_profile_activity_user_image_view"
+            app:layout_constraintRight_toRightOf="@+id/add_profile_activity_user_image_view" />
+
+          <TextView
+            android:id="@+id/add_profile_activity_required_heading_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="32dp"
+            android:text="@string/add_profile_required"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="12sp"
+            app:layout_constraintTop_toBottomOf="@+id/add_profile_activity_user_image_view" />
+
+          <org.oppia.android.app.profile.ProfileInputView
+            android:id="@+id/add_profile_activity_user_name_profile_input_view"
+            android:layout_width="match_parent"
```

Comment:
Required because of above change

`android:layout_width="0dp"`

Reply:
Done